internal/task/task/bs/balance_cluster.go

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2022-04-22
+ * Author: Ye Tao (YhhHaa)
+ */
+
+package bs
+
+import (
+	"fmt"
+
+	"github.com/opencurve/curveadm/cli/cli"
+	"github.com/opencurve/curveadm/internal/configure/topology"
+	"github.com/opencurve/curveadm/internal/task/scripts"
+	"github.com/opencurve/curveadm/internal/task/step"
+	"github.com/opencurve/curveadm/internal/task/task"
+	tui "github.com/opencurve/curveadm/internal/tui/common"
+)
+
+// @title NewBalanceTask
+// @description balance leader rapidly
+// @auth 	yetao 		2022/04/21 15:01
+// @param 	curveadm 	*cli.CurveAdm 			curveadm struct pointer
+// @param	dc 			*topology.DeployConfig  topology configuration file content
+// @return  *task.Task	Task struct				all tasks
+// @return	error		error					return err if sth wrong happended
```

Comment:
it's not the comment style of `go` language, you can refer to the buit-in library of `go` language, like [strings.go](https://github.com/golang/go/blob/e7c56fe9948449a3710b36c22c02d57c215d1c10/src/strings/strings.go#L121).